cashier-google-play-billing-debug/src/main/java/com/getkeepsafe/cashier/billing/debug/FakeGooglePlayCheckoutActivity.java

```diff
@@ -0,0 +1,105 @@
+package com.getkeepsafe.cashier.billing.debug;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.text.SpannableString;
+import android.text.style.StyleSpan;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.android.billingclient.api.BillingClient;
+import com.android.billingclient.api.Purchase;
+import com.getkeepsafe.cashier.Product;
+import com.getkeepsafe.cashier.billing.GooglePlayBillingSecurity;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class FakeGooglePlayCheckoutActivity extends Activity {
+
+    private static final String ARGUMENT_PRODUCT = "product";
+    private static final String ARGUMENT_PACKAGE = "package";
+    private static final String ARGUMENT_DEV_PAYLOAD = "dev-payload";
+    private static final String ARGUMENT_PRIVATE_KEY = "private-key";
+
+    private Product product;
+    private String privateKey64;
+
+    public static Intent intent(Context context, Product product, String privateKey64) {
+        Intent intent = new Intent(context, FakeGooglePlayCheckoutActivity.class);
+        intent.putExtra(ARGUMENT_PRODUCT, product);
+        intent.putExtra(ARGUMENT_PRIVATE_KEY, privateKey64);
```

Comment:
I'm not sure if passing the `privateKey64` simulates how the signing happens with Google Play Billing. From my understanding, this private key is stored within Google server where the data is signed when making a purchase. I think a more clear simulation would be to create a constant class where public and private key are stored. Reference the private key internally for this class. Then add documentation to the public key field that the data should be copied from the Google developer console. Similar to [this](https://github.com/googlesamples/android-play-billing/blob/master/TrivialDrive_v2/shared-module/src/main/java/com/example/billingmodule/billing/BillingManager.java) L:79.

I think it would also be good to add to the README.

Reply:
The private and public keys are constants defined in FakeGooglePlayBillingAPI class. I just passed it using intent, but your're right it doesn't make much sense. I copied that from existing sample. I added javadoc to public key variable and description in README file.